google/cloud/storage/internal/object_streambuf_test.cc

```diff
@@ -341,6 +341,95 @@ TEST(ObjectWriteStreambufTest, CreatedForFinalizedUpload) {
   EXPECT_EQ(close_result.value().payload, payload);
 }
 
+/// @test Verify that last error status is accessible for small payload.
+TEST(ObjectWriteStreambufTest, erroneousStream) {
```

Comment:
I think the casing of this test name should be `ErroneousStream`.

Reply:
fixed.